skt/executable.py

```diff
@@ -220,68 +235,66 @@ def cmd_merge(cfg):
                                        merge_report_path, report_string)
 
                     return
-
                 report_string += '\n  - %s\n    into commit %s' % (mb[0],
                                                                    bhead[:12])
-                idx += 1
-
-        elif cfg.get('patch') or cfg.get('pw'):
-            report_string = 'We applied the following patch(es):'
-
-            if cfg.get('patch'):
-                for patch in cfg.get('patch'):
+                previous_merge = thing_to_merge
+                idx[0] += 1
+
+            elif thing_to_merge[0] == 'patch' or thing_to_merge[0] == 'pw':
+                if previous_merge[0] == 'merge_ref' or not previous_merge[0]:
```

Comment:
Similar reason as above (and readability), it would be more clear to see what's going on with a modified condition of `if previous_merge[0] not in ['patch', 'pw']`, what do you think? Without that index if we go with the suggestion above about the `previous_merge` variable

Reply:
I fix it